internal/router/router.go

```diff
@@ -286,6 +292,9 @@ func (r *Router) serveMeta(id *charm.Reference, w http.ResponseWriter, req *http
 }
 
 func (r *Router) serveMetaGet(id *charm.Reference, req *http.Request) (interface{}, error) {
+	if err := r.authorize(id, req); err != nil {
```

Comment:
perhaps leave a comment:
// TODO consider whether we might want the capability to
// have different permissions for different meta endpoints.

?